resources/scripts.js

```diff
@@ -125,112 +188,362 @@ function buttonPosition(loc) {
+    // To implement in next PR
+    switch (loc) {
+        case 1:
+            chipContainerElement.style.visibility = 'hidden';
+            break;
+        case 4:
+            chipContainerElement.style.visibility = 'visible';
+            chipContainerElement.style.left = '85px';
+            break;
+        case 5:
+            chipContainerElement.style.visibility = 'visible';
+            chipContainerElement.style.left = '315px';
+            break;
+        case 6:
+            chipContainerElement.style.visibility = 'visible';
+            chipContainerElement.style.left = '545px';
+            break;
+        case 8:
+            chipContainerElement.style.visibility = 'visible';
+            chipContainerElement.style.left = '775px';
+            break;
+        case 9:
+            chipContainerElement.style.visibility = 'visible';
+            chipContainerElement.style.left = '1005px';
+            break;
+        case 10:
+            chipContainerElement.style.visibility = 'visible';
+            chipContainerElement.style.left = '1235px';
+            break;
+    }
+    switch (true) {
+        case (color > 0):
+            // White
+            break;
+        case (color >= 5):
+            // Red
+            break;
+        case (color >= 25):
+            // Green
+            break;
+        case (color >= 100):
+            // Black
+            break;
+        case (color >= 500):
+            // Orange
+            break;
+    }
+    return;
 }
 
-function playerLose() {
-    displayMessage('You lose')
+function payOut(isprimary, win, amount) {
+    if (isprimary === false) {
+        if (win === true) {
+            playerMoney += (amount * 2);
+            totalBets -= amount;
+            return;
+        }
+        totalBets -= amount;
+        return;
+    }
+    if (win === true) {
+        buildSummary(PRIMARYWIN);
+        lastPlayerMoney = playerMoney;
+        playerMoney += (amount * 2);
+        pushSecondaryBets();
+        resetGame();
+        moneyChange(0);
+        return;
+    }
+    buildSummary(PRIMARYLOSE);
     if (playerMoney === 0) {
         gameOver();
         return;
     }
-    buttonPosition(1);
-    betAmount = 0;
-    rollButtonState(false);
-    betButtonState(true);
+    displayMessage(rollSummary);
+    rollSummary = null;
+    buttonPosition(playerRoll);
+    resetGame();
+    moneyChange(0);
+    return;
 }
 
-function placeBet() {
-    betAmount = betDialog.value;
+function makeBet() {
+    betAmount = betDialogElement.value;
     betAmount = parseInt(betAmount);
-    if (validateBet() === false) {
+    if (validateBet(betAmount) === false) {
         return;
     }
+    lastPlayerMoney = playerMoney;
     playerMoney -= betAmount;
-    displayMoney.innerHTML = playerMoney;
-    rollButtonState(true);
-    betButtonState(false);
+    moneyChange(betAmount);
+    buttonStates();
     return;
 }
 
+function makeCome() {
+    let comeAmount = betDialogElement.value;
```

Comment:
you can combine this line and the bottom one and make comeAmount a `const`

Reply:
Changed to single `const` which parses int